packages/storybook-builder-vite/code-generator-plugin.js

```diff
@@ -1,10 +1,12 @@
-const fs = require('fs');
 const path = require('path');
 const { transformIframeHtml } = require('./transform-iframe-html');
 const { generateIframeScriptCode } = require('./codegen-iframe-script');
+const { normalizePath } = require('vite');
 
 module.exports.codeGeneratorPlugin = function codeGeneratorPlugin(options) {
     const virtualFileId = '/virtual:/@storybook/builder-vite/vite-app.js';
+    const iframeAbsoluteFileName = path.resolve(__dirname, 'input', 'iframe.html')
+    const iframeRelativeFileName = normalizePath(path.relative(process.cwd(), iframeAbsoluteFileName))
```

Comment:
Can we prevent `process.cwd()` usage here? @shilman I think it's a design choice in Storybook that directories are configurable, and not based on working directories?

Reply:
Yes there's a `--configDir` flag or something like that